go-controller/pkg/node/default_node_network_controller.go

```diff
@@ -872,7 +872,7 @@ func (nc *DefaultNodeNetworkController) Start(ctx context.Context) error {
 						nodeSubnets, err := util.ParseNodeHostSubnetAnnotation(&node, types.DefaultNetworkName)
 						if err != nil {
 							err1 = fmt.Errorf("unable to fetch node-subnet annotation for node %s: err, %v", node.Name, err)
-							return false, nil
+							continue
```

Comment:
shouldn't the continue be done only if this annotation is "notFound" ? other types of errors should they be continued as well?

Reply:
also should we print a message saying you skipped this node because you did not find the annotation?

Reply:
not sure my update is what you really wanted. the only possible error that we could get from ParseNodeHostSubnetAnnotations was that it was missing, but I *think* have it very specifically matching on the error message for that now. any new error that comes down the road wouldn't end up continuing.

Reply:
@tssurya , with my latest update we are going to spam the log message until the node gets it's annotation. i know this is a corner case, but until the CIDR is expanded we will get a lot of them...

but, I did test these current changes locally and they seem to work as expected